libs/utils/android/workloads/youtube.py

```diff
@@ -56,6 +56,8 @@ def run(self, exp_dir, video_url, video_duration_s, collect=''):
         # Force screen in LANDSCAPE mode
         Screen.set_orientation(self.target, portrait=False)
 
+        self.target.execute('monkey -p {} 1'.format(self.package))
```

Comment:
I used it because it allowed me to start YouTube without playing any video and then reset the frame stats. We can add it to the API, but I'm not sure if we are going to use for other workloads.